src/commands/Timer.ts

```diff
@@ -1,5 +1,6 @@
 
 import * as vscode from 'vscode';
+const schedule = require('node-schedule');
```

Comment:
typescript 内なので、`import` 構文使えませんでしたか？

```suggestion
import * as schedule from 'node-schedule';
// または必要なモジュールだけロード
import { scheduleJob }  from 'node-schedule';
```

Reply:
https://github.com/naruhiyo/appetizer/pull/4/commits/4b23d936b91758f73f5a522d773fe2c530ff6d63 で対応しました！